app/code/Magento/Customer/Observer/AfterAddressSaveObserver.php

```diff
@@ -91,6 +105,8 @@ class AfterAddressSaveObserver implements ObserverInterface
      */
     public function __construct(
         Vat $customerVat,
+        DateTimeFactory $dateFactory,
+        CustomerFactory $customerFactory,
```

Comment:
Hi, @arijit-de, due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't add required params to the constructor.

Reply:
@VladimirZaets Ok, I just need to check to get a work around